src/openforms/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -4,7 +4,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Open Forms\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2024-03-21 09:55+0100\n"
```

Comment:
Maybe we could add a check in CI to see if `makemessages/compilemessages` were run?

Reply:
I considered it, but I don't think it's reliable because gettext/rosetta use different line lengths and you end up with diffs that aren't actually changes, which can lead to false positives :/